virt_lightning/shell.py

```diff
@@ -228,6 +228,14 @@ def main():
         default=True,
     )
     start_parser.add_argument("distro", help="Name of the distro", type=str)
+    start_parser.add_argument(
+        "--disk",
+        action="extend",
+        nargs="*",
```

Comment:
I've afraid that if `--disk` accepts several values we will generate more confusion than good.

Can you please set `nargs=1` this way we accept only one value per `--disk`. Otherwise, everything looks perfect. Thank you for the patch.